src/ewmh_m2m/window.py

```diff
@@ -34,16 +34,11 @@ def fullscreen(self) -> bool:
         return '_NET_WM_STATE_FULLSCREEN' in state
     
     @fullscreen.setter
-    def fullscreen(self, state: bool):
+    def fullscreen(self, state: int):
         atom = xpybutil.util.get_atom
-        if state:
-            xpybutil.ewmh.request_wm_state(
-                self.window, 1, atom('_NET_WM_STATE_FULLSCREEN')
-            )
-        else:
-            xpybutil.ewmh.request_wm_state(
-                self.window, 0, atom('_NET_WM_STATE_FULLSCREEN')
-            )
+        xpybutil.ewmh.request_wm_state(
+            self.window, state, atom('_NET_WM_STATE_FULLSCREEN')
+        )
```

Comment:
Sorry, I was not clear in my review. Using a boolean value is very fine (it makes sense semantically, more than an int), but the conditionnal logic can be simplified, as shown below.
Thanks for the contribution.

```suggestion
    def fullscreen(self, state: bool):
        xpybutil.ewmh.request_wm_state(
            self.window, int(state), xpybutil.util.get_atom('_NET_WM_STATE_FULLSCREEN')
        )
```